lte/gateway/python/magma/enodebd/enodeb_status.py

```diff
@@ -513,80 +553,82 @@ def _get_gps_status_as_bool(enodeb: EnodebAcsStateMachine) -> bool:
+        return lat, lon, ait
     except (KeyError, ConfigurationError):
         return _get_cached_gps_coords()
     except ValueError:
-        logger.warning('GPS lat/long not understood (%s/%s)', lat, lon)
-        return '0', '0'
+        logger.warning('GPS lat/long/ait not understood (%s/%s/%s)', lat, lon, ait)
+        return '0', '0', '0'
 
 
-def _get_cached_gps_coords() -> Tuple[str, str]:
+def _get_cached_gps_coords() -> Tuple[str, str, str]:
     """
     Returns cached GPS coordinates if enB is disconnected or otherwise not
     reporting coordinates.
 
     Returns:
-        (str, str): (GPS lat, GPS lon)
+        (str, str, str): (GPS lat, GPS lon, GPS ait)
     """
     # pylint: disable=global-statement
-    global _gps_lat_cached, _gps_lon_cached
-    if _gps_lat_cached is None or _gps_lon_cached is None:
-        _gps_lat_cached, _gps_lon_cached = _read_gps_coords_from_file()
-    return _gps_lat_cached, _gps_lon_cached
+    global _gps_lat_cached, _gps_lon_cached, _gps_ait_cached
+    if _gps_lat_cached is None or _gps_lon_cached is None or _gps_ait_cached is None:
+        _gps_lat_cached, _gps_lon_cached, _gps_ait_cached = _read_gps_coords_from_file()
+    return _gps_lat_cached, _gps_lon_cached, _gps_ait_cached
 
 
 def _read_gps_coords_from_file():
     try:
         with open(CACHED_GPS_COORD_FILE_PATH, encoding="utf-8") as f:
             lines = f.readlines()
-            if len(lines) != 2:
+            if len(lines) != 3:
                 logger.warning(
-                    'Expected to find 2 lines in GPS '
+                    'Expected to find 3 lines in GPS '
                     'coordinate file but only found %d',
                     len(lines),
                 )
-                return '0', '0'
+                return '0', '0', '0'
             return tuple(map(lambda l: l.strip(), lines))
     except OSError:
         logger.warning('Could not open cached GPS coordinate file')
-        return '0', '0'
+        return '0', '0', '0'
 
 
-def _cache_new_gps_coords(gps_lat, gps_lon):
+def _cache_new_gps_coords(gps_lat, gps_lon, gps_ait):
     """
     Cache GPS coordinates in the module-level variables here and write them
     to a managed file on disk.
 
     Args:
         gps_lat (str): latitude as a string
         gps_lon (str): longitude as a string
+        gps_ait (str): aititude as a string
     """
     # pylint: disable=global-statement
-    global _gps_lat_cached, _gps_lon_cached
-    _gps_lat_cached, _gps_lon_cached = gps_lat, gps_lon
-    _write_gps_coords_to_file(gps_lat, gps_lon)
+    global _gps_lat_cached, _gps_lon_cached, _gps_ait_cached
+    _gps_lat_cached, _gps_lon_cached, _gps_ait_cached = gps_lat, gps_lon, gps_ait
+    _write_gps_coords_to_file(gps_lat, gps_lon, gps_ait)
 
 
-def _write_gps_coords_to_file(gps_lat, gps_lon):
-    lines = '{lat}\n{lon}'.format(lat=gps_lat, lon=gps_lon)
+def _write_gps_coords_to_file(gps_lat, gps_lon, gps_ait):
```

Comment:
```suggestion
def _write_gps_coords_to_file(gps_lat: str, gps_lon: str, gps_ait: str):
```

Reply:
![image](https://user-images.githubusercontent.com/87301024/147812405-49608eec-ec7a-4a7c-b489-c6596b2b3d38.png)
modified,thank you!